normandy/base/tests/api/test_authentication.py

```diff
@@ -0,0 +1,159 @@
+import json
+import pytest
+
+from rest_framework import permissions
+from rest_framework.response import Response
+from rest_framework.test import APIClient
+from rest_framework.views import APIView
+
+from django.conf.urls import url
+
+from normandy.base.tests import UserFactory
+from normandy.base.api.authentication import BearerTokenAuthentication
+
+
+class MockView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def make_response(self, request):
+        return Response({'user': request.user.email})
+
+    def get(self, request):
+        return self.make_response(request)
+
+    def post(self, request):
+        return self.make_response(request)
+
+    def put(self, request):
+        return self.make_response(request)
+
+
+urlpatterns = [
+    url(
+        r'^bearer/$',
+        MockView.as_view(authentication_classes=[BearerTokenAuthentication])
+    ),
+]
+
+
+@pytest.mark.django_db
+@pytest.mark.urls('normandy.base.tests.api.test_authentication')
+class TestBearerTokenAuthentication(object):
+    """Bearer token authentication"""
+
+    @pytest.fixture
+    def csrf_api_client(self):
+        return APIClient(enforce_csrf_checks=True)
+
+    @pytest.fixture
+    def mock_oidc(self, settings, requestsmock):
+        def fn(user=None, **kwargs):
+            if user:  # If content kwarg is not empty this user will be ignored
```

Comment:
I don't really understand this comment. Is it still correct?

Reply:
It's not still correct. I'll get rid of that.